lib/src/cmcd/CmcdV2Object.ts

```diff
@@ -0,0 +1,354 @@
+import type { CmcdCustomKey } from './CmcdCustomKey';
+import type { CmcdValue } from './CmcdValue';
+import type { CmcdStreamingFormat } from './CmcdStreamingFormat';
+import type { CmcdStreamType } from './CmcdStreamType';
+import type { CmcdPlayerState } from './CmcdPlayerState';
+
+export const CMCD_V2_COMMON_KEYS = [
+	'br', 
+	'ab', 
+	'bl', 
+	'tbl', 
+	'bs', 
+	'cdn', 
+	'cid', 
+	'ltc', 
+	'mtp', 
+	'pr', 
+	'sf', 
+	'sid', 
+	'bg', 
+	'sta', 
+	'st', 
+	'ts', 
+	'tpb', 
+	'tb', 
+	'lb', 
+	'tab', 
+	'lab', 
+	'pt', 
+	'ec', 
+	'msd', 
+	'v',
+] as const;
+
+/**
+ * Common Media Client Data (CMCD) is a standardized set of HTTP request header fields and query string parameters.
+ *
+ * @group CMCD
+ *
+ * @beta
+ */
+export type CmcdV2Object = {
```

Comment:
Instead of repeating all of the V1 keys and docs, wouldn't it be better to extend the V1 object? We also need to rethink the name of this type. We don't include data suffixes like "Object" in type names in CML. I see there is already a `CmcdV2` in this PR, so I'll have to more thoroughly review what each type is meant to represent.

```suggestion
export type CmcdV2Object = Cmcd & {
```

Reply:
Done